blazor/pdfviewer-2/how-to/deploy-maui-using-android-emulator.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: Deploy SfPdfViewer in Blazor MAUI in windows | Syncfusion
+description: Learn how to deploy SfPdfViewer in Blazor MAUI Application on Windows in Syncfusion Blazor SfPdfViewer component and much more details.
+platform: Blazor
+control: SfPdfViewer
+documentation: ug
+---
+
+# Render PDF Viewer Component from embedded source in the MAUI app 
```

Comment:
Can we modify this title "**Render PDF Document from embedded source in the MAUI app**"